src/components/CardList.js

```diff
@@ -0,0 +1,23 @@
+import '../styles/CardList.css';
+import Card from '../components/Card';
+
+const CardList = ({ folderInfo }) => {
+  const cardList = folderInfo?.folder.links;
+  console.log(cardList);
+
+  return (
+    <div className="cardlist">
+      {cardList ? (
+        <>
+          {cardList.map((cardlist) => (
+            <Card cardlist={cardlist} key={cardlist.id} />
+          ))}
```

Comment:
배열 `cardList`인 원소들을 map 하고 있는데 원소들의 이름을 `cardlist`로 L을 소문자로만 바꿔서 구분해주고 있습니다.
변수의 이름은 변수를 잘 나타내줘야하는데 이 경우에는 각 원소가 배열이 아니라 그냥 `card` 데이터죠? 
원소의 이름을 `card`로 바꿔야합니다.

```suggestion
          {cardList.map((card) => (
            <Card card={card} key={card.id} />
          ))}
```

Reply:
d2b06928a94e4648aaba03d17571cdd1b00950b9 props drilling 해결하면서 해당 부분도 수정했습니다~!